index.js

```diff
@@ -67,20 +68,18 @@ class HookShellScriptPlugin {
    * @param {string} msg
    */
   _handleError(msg) {
-    msg = `\n[${NAME}] ${msg}\n`;
     if (!this.watch) {
-      throw new Error(msg);
+      throw new Error(`[${NAME}] ${msg}`);
```

Comment:
Kept `NAME` in the error, but removed from the prefix of the logger message, as Webpack will prepend the message with `[${NAME}] `.